sql/arraytype.go

```diff
@@ -129,9 +129,19 @@ func (t arrayType) SQL(v interface{}) (sqltypes.Value, error) {
 		return sqltypes.Value{}, err
 	}
 
-	val, err := json.Marshal(v)
-	if err != nil {
-		return sqltypes.Value{}, err
+	var val []byte
```

Comment:
Can we actually kill off this type? It's not actually supported in MySQL, so the semantics of it are very unclear. And it's getting all muddled in the with the JSON array logic. I'd rather have exactly one well supported implementation of arrays (use JSON).

Also kill off the EXPLODE function at the same time.

Could be a follow up after this PR to keep things smaller.